Plugin/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Omise\Payment\Plugin;
+
+class CsrfValidatorSkip
+{
+    protected $urlInterface;
+    public function __construct(\Magento\Framework\UrlInterface $urlInterface) {
+        $this->urlInterface = $urlInterface;
+    }
+    /**
+     * @param \Magento\Framework\App\Request\CsrfValidator $subject
+     * @param \Closure $proceed
+     * @param \Magento\Framework\App\RequestInterface $request
+     * @param \Magento\Framework\App\ActionInterface $action
+     */
+    public function aroundValidate(
+        $subject,
+        \Closure $proceed,
+        $request,
+        $action
+    ) {
+        if ($request->getModuleName() == 'omise' && strpos($this->urlInterface->getCurrentUrl(), 'omise/callback/webhook')) {
```

Comment:
also minor, but speaking on behalf of @jonrandy (he didn't say that, but, I got a feeling lol)
"how about having a constant variable on `Omise\Payment\Block\Adminhtml\System\Config\Form\Field\Webhook` class then refer it here". Just to avoid any potential of forgetting to update the url as it exposes in 2 places.

```php
use Omise\Payment\Block\Adminhtml\System\Config\Form\Field\Webhook;

if (... && strpos($this->urlInterface->getCurrentUrl(), Webhook::URI)) { ... }
```
Maybe 😂

Reply:
Agree